ansible/zapd_cron.yml

```diff
@@ -4,6 +4,14 @@
   become: yes
   become_user: root
 
+  vars:
+   dest_email: alerts@zap.me
```

Comment:
ansible_deploy.sh sets a var (ADMIN_EMAIL=admin@zap.me) add one also there for alerts@zap.me